gstools/transform/field.py

```diff
@@ -335,12 +335,12 @@ def _zinnharvey(field, conn="high", mean=None, var=None):
         mean = np.mean(field)
     if var is None:
         var = np.var(field)
-    field = np.abs((field - mean) / np.sqrt(var))
+    field = np.abs((field - mean) / var)
     field = 2 * erf(field / np.sqrt(2)) - 1
     field = np.sqrt(2) * erfinv(field)
     if conn == "high":
         field = -field
-    return field * np.sqrt(var) + mean
```

Comment:
Why reverting this? I fixed just it in 0f24aad

I think the std-deviation needs to used there